src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableConnectionHelper.cs

```diff
@@ -763,7 +763,7 @@ public static ServerHostInfo GetServerHostInfo(IDbConnection connection)
         /// Gets the server host cpu count and memory from sys.dm_os_sys_info view
         /// </summary>
         /// <param name="connection">The connection</param>
-        public static ServerSystemInfo GetServerSystemInfo(IDbConnection connection)
+        public static ServerSystemInfo GetServerCpuAndMemoryInfo(IDbConnection connection)
         {
             var sysInfo = new ServerSystemInfo();
```

Comment:
It looks like this just has these values be ints - would suggest making them nullable so that they get sent over as undefined if an error occurs fetching them.